browser/src/view/components/Navbar/NavigationMenu.js

```diff
@@ -1,31 +1,32 @@
 import React from 'react';
-import { FiHome, FiLayers, FiSettings, FiLogOut, FiCircle } from 'react-icons/fi';
-import './NavigationMenu.css';
+import PropTypes from 'prop-types';
+import './NavigationMenu.scss';
 
-export const NavigationMenu = () => {
+export const NavigationMenu = ({ links }) => {
   return (
-    <>
-      <div className='navbar'>
-        <div className='nav-top'>
-          <div className='icon-container'>
-            <FiCircle className='nav-icon logo' />
-          </div>
-
-          <div className='icon-container'>
-            <FiHome className='nav-icon' />
-          </div>
-
-          <div className='icon-container'>
-            <FiLayers className='nav-icon' />
-          </div>
-        </div>
-        <div className='nav-bottom'>
-          <div className='icon-container'>
-            <FiSettings className='nav-icon' />
-          </div>
-          <div className='avatar'></div>
-        </div>
-      </div>
-    </>
+    <nav className='navbar'>
+      <ul className='nav-section-one'>
```

Comment:
Remember what I mentioned about intent.  Does `nav-section-one` have any design or user value?  Is there something thematic that ties these items together beyond them being section one?

Reply:
Sounds good. We spoke about this offline, but to just echo what we spoke about here for record-keeping, I'll change these sections to be more relevant to what the intent of each section is doing, e.g. something like 'user-actions' and 'user-settings' perhaps.